assets/js/pad.js

```diff
@@ -0,0 +1,19 @@
+function Pad() {
+    
+    this.sketch;
```

Comment:
how do you set this Pad's sketch? should be a parameter for Pad, or there should be a function that allows you to access it.